src/routers/adminRouter.js

```diff
@@ -0,0 +1,8 @@
+import express from 'express';
+import { getUsers } from '../controllers/users';
+
+const adminRouter = express.Router();
+
+adminRouter.get('/users', getUsers);
```

Comment:
I created a GET users route because the admin platform would need to get all the users in the database. This will allow an admin to change the user roles as needed.